packages/core/src/authMetadata.ts

```diff
@@ -17,10 +17,11 @@ class AuthMetadata implements IAuthMetadata {
   }
 
   static fromJSON(value: StringifiedType): AuthMetadata {
-    const { data, sig } = value;
+    // need to inject legacyMetadata flag
```

Comment:
is this comment a TODO? or can be removed?

Reply:
not a todo.
It remind that for tkeyTss deserialization, legacyMetadata Flag needed to be injected as we are trying to force upgrade the metadata format to v1.0.0 from v0.0.1 when the legacyMetadataFlag is false

Reply:
not sure what this means. where do you inject the flag?

Reply:
here
https://github.com/tkey/tkey/blob/35dc228c98ab8f631189ca87f5a7ecbcb4024b08/packages/core/src/core.ts#L938